features-linux.md

```diff
@@ -150,6 +150,7 @@ Irrelevant to the kernel version of the host operating system.
     "SCMP_ARCH_PPC",
     "SCMP_ARCH_PPC64",
     "SCMP_ARCH_PPC64LE",
+    "SCMP_ARCH_RISCV64",
```

Comment:
Not every implementation supports RISC-V, so this change is unnecessary

Reply:
Done - removed.